.github/workflows/dependency-review.yaml

```
@@ -29,4 +30,5 @@ jobs:
       - name: 'Dependency Review'
         uses: actions/dependency-review-action@11310527b429536e263dc6cc47873e608189ba21
```

If you're updating things, mind changing this?

```suggestion
        uses: actions/dependency-review-action@v3
```

---

Committed.  I wondered why we used the hash here but not anywhere else.